src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -530,6 +616,36 @@ subroutine init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, soilstate_i
       is_cellulose(i_soil3) = .false.
       is_lignin(i_soil3) = .false.
 
+      floating_cn_ratio_decomp_pools(i_micr1) = .true.
```

Comment:
These new paragraphs follow the prev. paragraphs as a template. Things may need to be corrected.

Reply:
yes, we can go through this if needed